### PR TITLE
add Enable validation messages for resource status tag for helm template

### PR DESCRIPTION
**Pull request by carolynhu**
add set label of GALLEY_ANALYSIS to test the overhead of enable validation messages for resource status tag

### CODE REVIEW

File: perf/benchmark/setup_test.sh [hunk 1]
```diff
@@ -54,6 +55,7 @@ function run_test() {
       --set server.inject="${ISTIO_INJECT}"  \
       --set client.injectL="${LINKERD_INJECT}" \
       --set server.injectL="${LINKERD_INJECT}" \
+      --set values.galley.enableAnalysis="${GALLEY_ANALYSIS}" \
```
Comment (howardjohn) [inline perf/benchmark/setup_test.sh]: This call to helm is for the templates not the istio install?

Comment (carolynhu) [reply perf/benchmark/setup_test.sh]: oh, yeah, sorry, need to rename this PR

**Comment by carolynhu** [comment]
I'm going to close this PR, since we cannot test it in our data plane test cases. We should design a separate control plane test for this.

**Comment by howardjohn** [comment]
I have some WIP work on control plane testing

**Comment by carolynhu** [comment]
@howardjohn I add you to tomorrow's sync up meeting.

**Pull request closed without merging**